File: Modeling/app/src/main/java/pisarev/com/modeling/interfaces/Drawing.java
package pisarev.com.modeling.interfaces;

import android.graphics.Canvas;
import pisarev.com.modeling.mvp.model.MyData;
import pisarev.com.modeling.mvp.model.Point;

public interface Drawing {
    void drawContour(Canvas canvas, MyData data, Point pointCoordinateZero, float zoom, int index);
    void setNumberLine(int numberLine);
}

File: Modeling/app/src/main/java/pisarev/com/modeling/interfaces/Drawing.kt
package pisarev.com.modeling.interfaces

import android.graphics.Canvas
import pisarev.com.modeling.mvp.model.core.MyData
import pisarev.com.modeling.mvp.contur.Point

interface Drawing {
    fun drawContour(canvas: Canvas, data: MyData, pointCoordinateZero: Point, zoom: Float, index: Int)
    fun setNumberLine(numberLine: Int?)
}